### DISCUSSION TITLE
채널 _id 상수로 저장해두고 사용할지 여부

### DISCUSSION THREAD

**Discussion by NamgyungKim** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#109] [Ideas]
채널 _id 상수로 저장해두고 사용할지 여부
이유: 채널별로 post내용불러오려면 {{baseURL}}/posts/channel/62a55de1c882bf3a287f961c 이런식으로 불러와야한다.
흠.. 근데 제 메인페이지에서만 사용되는 api인것도 같네요...
혹시 다른 분들 체널 id 필요한 페이지 있으신가요??

**Comment by YuHyun-P** [comment]
CHANNEL = { "front": "62a55de1c882bf3a287f961c", "back": "62a55de1c882bf3a287f961c", ... }

**Reply by Hyevvy** [reply]
const FRONT="62a55de1c882bf3a287f961c"

**Reply by YuHyun-P** [reply]
constants/channel.ts
CHANNELS.front

**Reply by YuHyun-P** [reply]
SKILL_OPTIONS

**Comment by Hyevvy** [comment]
SKILL_OPTIONS.CHANNEL

**Reply by Hyevvy** [reply]
SKILL_OPTIONS.options
SKILL_OPTIONS.channel

**Comment by YuHyun-P** [comment]
✅ constants 폴더에 camelCase.ts 파일로 생성 → 파일 내 상수 변수 이름은 대문자 + 언더바 케이스로! (Wiki 폴더 구조에 해당 내용 추가 하겠습니당)
